src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputSeparatorWrapArrayDeclarator.java

```diff
@@ -6,7 +6,7 @@ class InputSeparatorWrapArrayDeclarator {
             ] {1, 2};
 
     protected int[] arrayDeclarationWithBadWrapping = new int
-            [] {1, 2}; // warn
+            [] {1, 2}; // violation ''.*' should be on the previous line.'
```

Comment:
`''.*'`
Did you mean ',' ?

Not clear what we violating

Reply:
violation is for `[`, the violation is: 

```
// violation ''[' should be on the previous line.'
```

The `[` was causing mismatching in comparing violation messages so had to use `.*` in to prevent that

Reply:
Can you put `\` in front to escape such symbols, we need clarity on what we violating

Reply:
Done. 

https://github.com/checkstyle/checkstyle/blob/4d6e49aefdf4783b7bcc87000aa189a9603cc458/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputSeparatorWrapArrayDeclarator.java#L8-L9

thanks for the suggestion